test/junos/mocked_data/test_get_ipv6_neighbors_table/mac_is_none/get-ipv6-nd-information.xml

```diff
@@ -0,0 +1,12 @@
+<ipv6-nd-information>
+    <ipv6-nd-entry>
+        <ipv6-nd-neighbor-address>2001:0DB8::3</ipv6-nd-neighbor-address>
+        <ipv6-nd-neighbor-l2-address>none</ipv6-nd-neighbor-l2-address>
+        <ipv6-nd-state>unreachable</ipv6-nd-state>
+        <ipv6-nd-expire>3</ipv6-nd-expire>
+        <ipv6-nd-isrouter>no</ipv6-nd-isrouter>
+        <ipv6-nd-issecure>no</ipv6-nd-issecure>
+        <ipv6-nd-interface-name>ae0.3</ipv6-nd-interface-name>
+    </ipv6-nd-entry>
+    <ipv6-nd-total>1</ipv6-nd-total>
+</ipv6-nd-information>
```

Comment:
minor nit: missing newline at end

Reply:
> minor nit: missing newline at end

Funnily enough my IDE strips them on XML files. Had to do this in vim :laughing: